hugegraph-core/src/main/java/com/baidu/hugegraph/structure/HugeVertex.java

```diff
@@ -323,10 +323,7 @@ public HugeEdge constructEdge(String label, HugeVertex vertex,
         ElementHelper.attachProperties(edge, keyValues);
 
         // Set id if it not exists
-        if (id == null) {
-            edge.assignId();
-        }
-
+        edge.assignId();
```

Comment:
don't assignId if id exist

Reply:
but in the context, the id must be null, otherwise it throws exception.... seems confuse